src/Native/Windows/build-native.cmd

```diff
@@ -10,36 +10,53 @@ set __IntermediatesDir=""
 set __BuildArch=AnyCPU
 set __VCBuildArch=x86_amd64
 set CMAKE_BUILD_TYPE=Release
+set __EnvironmentSet=false
+set __VSVersion=vs2013
```

Comment:
> The CoreFX and CoreCLR windows native builds used to use the exact same code

As far as I know, it was never the case. There was always repo-specific code for the native builds, but the stuff that could have been same was actually same.